### PR TITLE
Add Spiral Matrix Exercise

### PR DESCRIPTION
**Pull request by tomasnorre**
I was just in the flow. Hope that's OK? I'll post a link in the forum too. 

https://forum.exercism.org/t/implement-some-missing-php-exercises-for-48in24/9865

### CODE REVIEW

**Review by neenjaw — APPROVED** [review]
LGTM

File: exercises/practice/spiral-matrix/.meta/example.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+ * <link>.
+ *
+ * To disable strict typing, comment out the directive below.
+ */
+
+declare(strict_types=1);
+
+class SpiralMatrix
+{
+    public function draw($n): array
```
Comment (mk-mxp) [inline exercises/practice/spiral-matrix/.meta/example.php:29]:
```suggestion
    public function draw(int $n): array
```

File: exercises/practice/spiral-matrix/SpiralMatrix.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ * <link>.
+ *
+ * To disable strict typing, comment out the directive below.
+ */
+
+declare(strict_types=1);
+
+class SpiralMatrix
+{
+    public function draw($n): array
```
Comment (mk-mxp) [inline exercises/practice/spiral-matrix/SpiralMatrix.php:29]:
```suggestion
    public function draw(int $n): array
```

**Review by mk-mxp — APPROVED** [review]
These are both optional. Resolve the threads, if you want me to merge it as is.

File: exercises/practice/spiral-matrix/.meta/example.php (hunk 1 above)
Comment (tomasnorre) [reply exercises/practice/spiral-matrix/.meta/example.php]: I have added the type for both. Like it better that way, but completely overlooked it.

**Pull request merged**